x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobTestUtils.java

```diff
@@ -51,11 +51,11 @@ public static ConnectorSyncJob getRandomConnectorSyncJob() {
             .setCompletedAt(randomFrom(new Instant[] { null, randomInstantBetween(lowerBoundInstant, upperBoundInstant) }))
             .setConnector(ConnectorTestUtils.getRandomSyncJobConnectorInfo())
             .setCreatedAt(randomInstantBetween(lowerBoundInstant, upperBoundInstant))
-            .setDeletedDocumentCount(randomLong())
+            .setDeletedDocumentCount(randomInt())
```

Comment:
Why change from long to int?

Reply:
This is supposed to be `integer`, so I've corrected it here. See [Connector protocol](https://github.com/elastic/connectors/blob/main/docs/CONNECTOR_PROTOCOL.md#elasticsearch-mappings-for-elastic-connectors-sync-jobs)

Reply:
I changed these to randomNonNegativeInt() after your other comment.